ovos_plugin_manager/tts.py

```diff
@@ -190,19 +193,25 @@ def create(config=None):
         """
         tts_config = get_tts_config(config)
         tts_module = tts_config.get('module', 'dummy')
+        if tts_module in OVOSTTSFactory.MAPPINGS:
```

Comment:
let's flag all mappings for deprecation in next stable, those were just to allow existing classic mycroft configs to work, but it has been long enough for users to migrate